app.py

```diff
@@ -212,6 +212,26 @@ def check():
         return jsonify(claim_code.to_json())
     return abort(404)
 
+@app.route("/merchanttxlog", methods=["POST"])
+def merchanttxlog():
+    sig = request.headers.get("X-Signature")
+    content = request.json
+    api_key = content["api_key"]
+    nonce = content["nonce"]
+    token = content["token"]
+    walletaddr = content["walletaddr"]
+    amount = content["amount"]
+    txid = content["txid"]
+    txdirection = content["txdirection"]
```

Comment:
change to just 'direction'

Reply:
problem:
@app.route("/merchanttx", methods=["POST"])
def merchanttx():
    sig = request.headers.get("X-Signature")
    content = request.json
    api_key = content["api_key"]
    nonce = content["nonce"]
    token = content["token"]
    walletaddr = content["walletaddr"]
    amount = content["amount"]
    txid = content["txid"]
    direction = content["direction"]
    type = content["type"]
    res, reason, api_key = check_auth(api_key, nonce, sig, request.data)
    if not res:
        return abort(400, reason)
    merchant_tx = MerchantTx(api_key.user, walletaddr, amount, txid, direction, type)
    db.session.add(merchanttx)
    db.session.commit()
    return jsonify(merchanttx.to_json())